Formula/google-protobuf.rb

```diff
@@ -59,6 +65,7 @@ def install
     ENV.universal_binary if build.universal?
     ENV.cxx11 if build.cxx11?
 
+    (buildpath/"gmock").install resource("gmock")
```

Comment:
@stanley-cheung sorry for the intrusion, the `.install` function is coming out of an overide that homebrew did on [Pathname.rb](https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/pathname.rb#L64) basicly they are creating the destanation path and copyting there the file that you wish to install.